quipucords/fingerprinter/task.py

```diff
@@ -1371,7 +1371,7 @@ def _process_vcenter_fact(self, source, fact):
         self._add_fact_to_fingerprint(source, 'vm.dns_name', fact,
                                       'vm_dns_name', fingerprint)
         self._add_fact_to_fingerprint(source, 'vm.host.name',
```

Comment:
Just out of curiosity why do we still have the `vm.host.name` in the fingerprinter here?

Reply:
It's a mapping between details report names and deployment report names.